test/regress/Makefile.tests

```diff
@@ -787,6 +787,12 @@ REG0_TESTS = \
 	regress0/strings/substr-rewrites.smt2 \
 	regress0/strings/type001.smt2 \
 	regress0/strings/unsound-0908.smt2 \
+	regress0/sym/sym1.smt2 \
```

Comment:
Can you move these to regress1/? Also, you should enable your option on each of these regressions via the annotation COMMAND-LINE: --symmetry-detect.
See other regressions for details.